os/gcron/gcron_cron.go

```diff
@@ -184,6 +186,7 @@ func (c *Cron) Stop(name ...string) {
 		}
 	} else {
 		c.status.Set(StatusStopped)
+		c.jobWaiter.Wait()
```

Comment:
It would be better change the parameter of function `Stop` to `StopOption` which would be defined as follows:
```go
type StopOption struct {
    JobName  string // Stops certain job by name.
    Graceful bool   // Blocks and waits all current running jobs done.
}

func (c *Cron) Stop(option StopOption) 
```

Reply:
> The original definition of the function [`Stop`](https://github.com/gogf/gf/blob/753965b9a95085094dee0059f679ecdc6d7e5eac/os/gcron/gcron_cron.go#L178) is as follows:
> 
> ```
> func (c *Cron) Stop(name ...string) {}
> ```
> 
> It seems that it cannot be changed to the above definition.
> 
> Is it possible to add a new `GracefulStop` function?
> 
> ```
> // GracefulStop Blocks and waits all current running jobs done.
> func (c *Cron) GracefulStop() {
> 	c.status.Set(StatusStopped)
> 	c.jobWaiter.Wait()
> }
> ```

There would be two `Stop` function entries, which would not be convenient understanding and using for developers. It might be better update the current `Stop` function directly, although there might be compatibility issue.